vue.json

```diff
@@ -0,0 +1,57 @@
+{
+  "extends": ["plugin:vue/vue3-essential"],
+  "parserOptions": {
+    "extraFileExtensions": [".vue"]
+  },
+  "plugins": ["vue"],
+  "rules": {
+    "vue/attributes-order": "error",
+    "vue/component-tags-order": "error",
+    "vue/no-lone-template": "error",
+    "vue/no-multiple-slot-args": "error",
+    "vue/no-v-html": "error",
+    "vue/order-in-components": "error",
+    "vue/this-in-template": "error",
+    "vue/attribute-hyphenation": "error",
+    "vue/component-definition-name-casing": "error",
+    "vue/first-attribute-linebreak": "error",
+    "vue/html-closing-bracket-spacing": "error",
+    "vue/html-end-tags": "error",
+    "vue/multiline-html-element-content-newline": "error",
+    "vue/mustache-interpolation-spacing": "error",
+    "vue/no-multi-spaces": "error",
+    "vue/no-spaces-around-equal-signs-in-attribute": "error",
+    "vue/no-template-shadow": "error",
+    "vue/one-component-per-file": "error",
+    "vue/prop-name-casing": "error",
+    "vue/require-default-prop": "error",
+    "vue/require-explicit-emits": "error",
+    "vue/require-prop-types": "error",
+    "vue/v-bind-style": "error",
+    "vue/v-on-event-hyphenation": "error",
+    "vue/v-on-style": "error",
+    "vue/v-slot-style": "error",
+    "vue/html-quotes": "error",
+
+    "vue/html-self-closing": [
+      "error",
+      {
+        "html": {
+          "void": "always"
+        }
+      }
+    ],
+
+    "putout/putout": "off",
+    "max-len": "off",
+    "lines-around-comment": "off"
```

Comment:
`lines-around-comment` is configured in `prettier.json` in a way that's compatible with Prettier, so it should not be disabled.

Reply:
This too was confliciting with something vue-related. Will check if I can come up with another solution so this rule can stay enabled.

Reply:
I will work on these later today (Monday)!